wazo_calld/plugins/dial_mobile/bus_consume.py

```diff
@@ -54,10 +56,42 @@ def _on_refresh_token_deleted(self, event):
         self._service.on_mobile_refresh_token_deleted(event['user_uuid'])
 
     def _on_bridge_enter(self, event):
-        self._service.cancel_push_mobile(event['Uniqueid'])
+        if event['BridgeType'] != 'stasis':
+            # Ignoring unrelated bridges
+            return
```

Comment:
> Why are Stasis bridges ignored? CTI transfers are made via `stasis` bridges.

All bridges created by the `dial_mobile` application are of type `stasis`. I filter out all none `stasis` to avoid doing ARI HTTP requests later for unrelated bridges.